tests/helpers.py

```diff
@@ -1,90 +1,142 @@
 from pathlib import Path
+from urllib.parse import urlparse
+import boto3
 
-TEST_MODEL_NAME = '[PyTest] Test Model'
-TEST_MODEL_REFERENCE = '[PyTest] Test Model Reference'
-TEST_MODEL_RUN = '[PyTest] Test Model Run'
-TEST_DATASET_NAME = '[PyTest] Test Dataset'
-TEST_SLICE_NAME = '[PyTest] Test Slice'
+PRESIGN_EXPIRY_SECONDS = 60 * 60 * 24 * 2  # 2 days
+
+TEST_MODEL_NAME = "[PyTest] Test Model"
+TEST_MODEL_REFERENCE = "[PyTest] Test Model Reference"
+TEST_MODEL_RUN = "[PyTest] Test Model Run"
+TEST_DATASET_NAME = "[PyTest] Test Dataset"
+TEST_SLICE_NAME = "[PyTest] Test Slice"
 TEST_IMG_URLS = [
-    's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/6dd63871-831611a6.jpg',
-    's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/82c1005c-e2d1d94f.jpg',
-    's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/7f2e1814-6591087d.jpg',
-    's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/06924f46-1708b96f.jpg',
-    's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/89b42832-10d662f4.jpg',
+    "s3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/6dd63871-831611a6.jpg",
+    "s3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/82c1005c-e2d1d94f.jpg",
+    "s3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/7f2e1814-6591087d.jpg",
+    "s3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/06924f46-1708b96f.jpg",
+    "s3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/89b42832-10d662f4.jpg",
 ]
 
+
+def get_signed_url(url):
```

Comment:
adding these helper functions was the best way I could think of to make testing with semseg masks feasible (need to sign the URL before upload).  It's a bit clunky, and I'm open to suggestions for how to avoid signed urls in the future

Reply:
This works fine, though one option in the future is to just move some semseg masks to a public s3 bucket for testing purposes

Reply:
I'll ask the platform team if we're allowed to make public buckets